canvas_course_admin_tools/settings/base.py

```diff
@@ -273,6 +273,10 @@
 
 PERMISSION_ISITES_MIGRATION_IMPORT_FILES = 'im_import_files'
 
+SECTIONS_TOOL = {
```

Comment:
This already exists in the [MANAGE_SECTIONS key](https://github.com/Harvard-University-iCommons/canvas_course_admin_tools/pull/25/files#diff-560236b5c0fe7591e778e1d52519d0feL292).